content/pt-br/docs/setup/production-environment/container-runtimes.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Agentes de execução do Contêiner
+content_type: concept
+weight: 30
+---
+
+<!-- overview -->
+
+{{< note >}}
+Dockershim foi removido do projeto Kubernetes desde a versão 1.24. Para mais detalhes leia [Dockershim Removal FAQ](/blog/2022/02/17/dockershim-faq).
+{{< /note >}}
+
+Você precisa instalar um {{< glossary_tooltip text="agente de execução do contêiner" length="all"  term_id="container-runtime" >}} em cada nó do cluster para que os pods possam ser executados. Esta página descreve as tarefas e o que está envolvido para configurar os nós. 
+
+O Kubernetes 1.24 requer um tempo de execução que esteja em conformidade com {{< glossary_tooltip term_id="cri" length="all" text="Container Runtime Interface">}} (CRI).
+
+Para mais informações, veja [versão de suporte CRI](#cri-versions).
+
+Esta página fornece uma descrição de como utilizar múltiplos container runtime comuns com Kubernetes.
+
+- [containerd](#containerd)
+- [CRI-O](#cri-o)
+- [Docker Engine](#docker)
+- [Mirantis Container Runtime](#mcr)
+
+{{< note >}}
+As versões do Kubernetes anteriores ao v1.24 incluem uma integração direta com o Docker Engine, utilizando um componente chamado _dockershim_. Esta integração direta especial não faz mais parte do Kubernetes (a remoção foi anunciada como parte da versão v1.20). Leia [Check whether Dockershim removal affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-removal-affects-you/) para entender como esta remoção pode afetar você. Para aprender mais sobre migração utilizando docker, acesse o [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
+
+Se você estiver utilizando uma versão do Kubernetes diferente da v{{< skew currentVersion >}}, verifique a documentação da versão que você estiver utilizando.
+{{< /note >}}
+
+<!-- body -->
+
+## Pré-requisitos de instalação e configuração
+
+As etapas a seguir se aplicam como configurações comuns para nós do Kubernetes no Linux.
+
+Etapas podem ser puladas caso tenha certeza que particularidades de configuração não serão necessárias.
+
+Para mais informações, consulte a [Network Plugin Requirements](/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#network-plugin-requirements) ou a documentação específica para a execução de seu container.
```

Comment:
The Network Plugin Requirements content is currently translated and can be linked directly here (`/pt-br/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/#requisitos-de-plugins-de-rede`). The text should be updated to match that subtitle as well.

Reply:
The original sentence after "ou" was:
"the documentation for your specific container runtime".

This translation is a bit far from that.
I suggest using "a documentação do seu agente de execução de contêiner específico", or something to that effect.